sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## 1.1.0 (2020-08-11)
 
+### Changes since 1.0.*
+
+- With 1.1.0, new developer credentials are now available: `VisualStudioCodeCredential` and `AzureCLICredential`. These developer credentials may be used directly or indirectly as part of `DefaultAzureCredential`.
+- Added the ability to configured the Managed Identity client ID via the `DefaultAzureCredential` constructor options.
```

Comment:
Am guessing we added a new option to the options bag? Can we call out its name?
Also, `configured` -> `configure`
Also, in the next section, we refer to "user-assigned client Id". Why not do the same here?